janusgraph-core/src/main/java/org/janusgraph/diskstorage/locking/consistentkey/ConsistentKeyLocker.java

```diff
@@ -380,7 +380,8 @@ private WriteResult tryWriteLockOnce(StaticBuffer key, StaticBuffer del, StoreTr
         Throwable t = null;
         final Timer writeTimer = times.getTimer().start();
         StaticBuffer newLockCol = serializer.toLockCol(writeTimer.getStartTime(), rid, times);
-        Entry newLockEntry = StaticArrayEntry.of(newLockCol, zeroBuf);
+        StaticArrayEntry newLockEntry = (StaticArrayEntry) StaticArrayEntry.of(newLockCol, zeroBuf);
+        newLockEntry.setMetaData(EntryMetaData.TTL, (int) lockExpire.getSeconds());
```

Comment:
I think this can be a nice experimental feature because someone might prefer Database native TTL to the JanusGraph [StandardLockCleanerService](https://github.com/JanusGraph/janusgraph/blob/master/janusgraph-core/src/main/java/org/janusgraph/diskstorage/locking/consistentkey/StandardLockCleanerService.java) (which can be enabled by [storage.lock.clean-expired](https://docs.janusgraph.org/basics/configuration-reference/#storagelock)). By experimental, I mean this should be configurable, optional, and disabled by default. Btw I don't see the point of having this TTL and StandardLockCleanerService enabled at the same time since both do the same thing.

Reasons I think this should be optional and disabled by default:
1) Not every database supports TTL. If the user wants to use this feature, they should be aware that their data backend supports TTL.
2) Some database supports TTL but it does not work this way. For example, BerkeleyDB supports TTL but only at the hour granular level. If you set the TTL to a short period of time, say, 2 minutes, the JanusGraph-BerkeleyDB adapter will automatically raise the TTL to 1 hour. Users are very likely to be unaware of this and might wonder why "expired" locks are not cleaned. This should be clearly documented in the configuration option.
3) Some database like Cassandra does not recommend using a short TTL. I am not familiar with the TTL implementation of Cassandra, so I am not sure if using a TTL here is a good option.